test/oqs_test_endecode.c

```diff
@@ -175,6 +175,11 @@ static int test_oqs_encdec(const char *alg_name) {
         if (pkey == NULL)
             goto end;
 
+        if (!OBJ_sn2nid(alg_name)) {
+            printf("No OID registered for %s", alg_name);
```

Comment:
```suggestion
            printf("No OID registered for %s\n", alg_name);
```
A newline here makes the output less ugly :) (see https://github.com/open-quantum-safe/oqs-provider/actions/runs/10872524798/job/30167659926?pr=522#step:6:488)